src/_includes/partials/head/meta.html

```diff
@@ -28,4 +28,8 @@
 {% endif %}
 
 {%- comment -%} Canonical URL for SEO {%- endcomment -%}
-<link rel="canonical" href="{{ site.base_url }}{{ page.url | url }}" />
\ No newline at end of file
+<link rel="canonical" href="{{ site.base_url }}{{ page.url | url }}" />
+
+{% if .title == "Components" %}
+<meta name=”robots” content=”noindex”>
```

Comment:
```suggestion
<meta name="robots" content="noindex">
```
I think you have used a slightly different unicode quote or something here ? Shows double in the browser.

![Screenshot 2023-12-21 at 9 42 19 AM](https://github.com/CloudCannon/small-business-template-eleventy/assets/101225153/d0a6e1fd-a927-46dd-be19-3f9964a31314)

Reply:
And that's why we don't copy and paste from the internet, kids....